src/js/reducers/sourceContentUpdatesReducer.js

```diff
@@ -14,6 +17,11 @@ const SourceContentUpdatesReducer = (state = initialState, action) => {
       ...state,
       resources: [],
     };
+  case consts.SOURCE_CONTENT_UPDATED:
```

Comment:
Should we make the action type more explicit to what it is actually doing? For instance, in this case it is updating the source content update count.